src/test/kotlin/io/cloudflight/gradle/autoconfigure/springdocopenapi/SpringDocOpenApiConfigurePluginTest.kt

```diff
@@ -0,0 +1,23 @@
+package io.cloudflight.gradle.autoconfigure.springdocopenapi
+
+import io.cloudflight.gradle.autoconfigure.test.util.ProjectFixture
+import io.cloudflight.gradle.autoconfigure.test.util.useFixture
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+
+class SpringDocOpenApiConfigurePluginTest {
+
+    @Test
+    fun `the openapi document is created`(
```

Comment:
please add another test with our typical module structure (i.e. the skeleton), where we have an `-api`, `-service`, `-ui` and `-server` module